src/coreclr/jit/codegenriscv64.cpp

```diff
@@ -1762,7 +1759,9 @@ void CodeGen::genLclHeap(GenTree* tree)
         unsigned outgoingArgSpaceAligned = roundUp(compiler->lvaOutgoingArgSpaceSize, STACK_ALIGN);
         // assert((compiler->lvaOutgoingArgSpaceSize % STACK_ALIGN) == 0); // This must be true for the stack to remain
         //                                                                // aligned
-        genInstrWithConstant(INS_addi, EA_PTRSIZE, REG_SPBASE, REG_SPBASE, outgoingArgSpaceAligned, rsGetRsvdReg());
+        if (tempReg == REG_NA)
```

Comment:
It looks that at this point tempReg is always REG_NA.

Reply:
I think it's ok without an assertion, var declaration isn't that far away.